templates/assetsconfig.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "superset.fullname" . }}-assets
+  labels:
+    app: {{ include "superset.name" . }}
+    chart: {{ include "superset.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+binaryData:
+  {{- $assetsLines := (splitList "\n" (trim (include "superset.nestedAssets" .))) }}
+  {{- range $line := $assetsLines }}
+    {{- $path := (index (splitList ":" $line) 0) }}
+    {{- $data := (index (splitList ":" $line) 1) }}
+  "{{ print ($path | replace "/" "..") }}": "{{ print $data }}"
```

Comment:
ConfigMaps can be directly mounted as files but nested directory structure is not supported - you can't even use "/" in keys themselves.  As a substitute, we use ".." since "." is an allowed char and no filename should include "..".